parsl/channels/ssh/ssh.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, hostname, username=None, password=None, script_dir=None, envs
         self.password = password
         self.kwargs = kwargs
         self.script_dir = script_dir
+        self.host_auth = host_auth
 
         if host_auth:
```

Comment:
Should that be `if self.host_auth` then?

Reply:
(shrug)

It is logically equivalent either way, I'm not sure if there's a reason to always do one over the other? I guess using the attribute as you suggest is better in case you move the code to a different method? We seem to mix elsewhere in the code.

Reply:
@ZhuozhaoLi both ways are equivalent. Why don't we go with this unless there's something else we are missing.